fabtests/common/shared.c

```diff
@@ -2724,6 +2726,13 @@ void ft_parse_addr_opts(int op, char *optarg, struct ft_opts *opts)
 		else
 			opts->oob_port = default_oob_port;
 		break;
+	case 'E':
```

Comment:
Should the -b option fall through (like you had previously), so that setting sync also sets addr_exch?

Reply:
The depends on if the FT_OPT_OOB_SYNC flag implies address exchange or not. Currently it is implied so there is no need to add the FT_OPT_OOB_ADDR_EXCH flag to the "-b" option. If we decide to make these two flags orthogonal then the "-b" option should have the two flags set. There are a few tests that set the FI_OPT_OOB_SYNC flag directly. They may need also to be changed if we go this path. I am fine with either choice.

Reply:
This was my thought.  The -b option currently implies ADDR_EXCH, which is why I was thinking of falling through to make setting the option explicit.   From the view of the code, we keep the two flags disjoint.  It's the command line option that enables both OOB_SYNC and ADDR_EXCH.

And I also thought about the tests that are setting OOB_SYNC.  I agree those may need to be updated.  We can consider creating a new define, OOB_CTRL = OOB_SYNC | ADDR_EXCH, to mean use OOB for all control exchanges.  ?

Reply:
OOB_CTRL sounds a good idea. I will go ahead to make the changes including the affected tests.